### PR TITLE
Update Measured disk usage

### PR DESCRIPTION
**Pull request by curquiza**

### CODE REVIEW

File: reference/under_the_hood/storage.md [hunk 1]
```diff
@@ -58,10 +58,16 @@ This dataset is a JSON file of 8.6 MB and has 19,553 documents. When we index th
 
 | Raw JSON | MeiliSearch database size on disk | RAM usage | Virtual memory usage |
 | -------- | --------------------------------- | -------------------- | ------------------- |
-| 9.1 MB   | 320 MB                            | ≃ 426 MB             | 205 Gb (memory map) |
+| 9.1 MB   | 224 MB                            | ≃ 305 MB             | 205 Gb (memory map) |
```
Comment (guimachiavelli) [inline reference/under_the_hood/storage.md:61]: Just to be sure, this is still using the `movies.json` dataset, right?

File: reference/under_the_hood/storage.md [hunk 2]
```diff
@@ -58,10 +58,16 @@ This dataset is a JSON file of 8.6 MB and has 19,553 documents. When we index th
...
 
 > This means the database is using 426 MB of RAM and 122 MB of disk space. Note that [virtual memory](https://www.enterprisestorageforum.com/hardware/virtual-memory/) refers only to disk space allocated by your computer for MeiliSearch—it does not mean that it's actually in use by the database. See [Memory Usage](#memory-usage) for more details.
 
+::: warning
+
+These metrics can strongly change depending on the machine where your index.
```
Comment (guimachiavelli) [inline reference/under_the_hood/storage.md:67]: I'm not sure I understand what you mean here. Is it a disclaimer saying that if people run the same test on a machine with different stats (e.g. more/less RAM), they will probably see different numbers?

```suggestion
These metrics are highly dependent on the machine that is running MeiliSearch. Running this test in significantly underpowered machines is likely to give different results.
```

**Review by guimachiavelli — CHANGES_REQUESTED** [review]

File: reference/under_the_hood/storage.md (hunk 1 above)
Comment (curquiza) [reply reference/under_the_hood/storage.md:61]: Yes! Why?

Comment (guimachiavelli) [reply reference/under_the_hood/storage.md:61]: No reason, just wanted to confirm it =)

**Review by guimachiavelli — APPROVED** [review]
Looks great! Thanks 💝

**Comment by dbolkensteyn** [comment]
@curquiza Is the virtual memory truly at `205 Gb`? I just read the page on the doc and thought it must have been a typo.

**Comment by curquiza** [comment]
Hello @curquiza, no typo if I'm correct, the virtual memory, is just virtual, not the real amount of memory Meilisearch will require 😄

**Pull request merged**